packages/stargate/src/stargateclient.ts

```diff
@@ -83,6 +80,7 @@ export interface BroadcastTxSuccess {
   readonly transactionHash: string;
   readonly rawLog?: string;
   readonly data?: readonly MsgData[];
+  readonly gasUsed: number;
```

Comment:
Can we add gasWanted here as well? This should be trivial to get now and probably makes some user happy